pages/user/edit.vue

```diff
@@ -0,0 +1,368 @@
+<template>
+  <div id="user-profile-view">
+    <input
+      type="file"
+      class="hidden"
+      ref="imageUpload"
+      accept="image/png, image/jpeg"
+      @change="fileUploaded()"
+    />
+    <div id="user-view">
+      <!-- Profile Card -->
+      <VxCard class="mb-base" v-if="AuthUser && UserData && UserInfo">
+        <!-- Avatar -->
+
+        <div class="vx-row w-full" style>
+          <div class="vx-row w-full lg:w-1/2">
+            <!-- Avatar Col -->
+            <div class="vx-col" id="avatar-col">
+              <div class="img-container mb-4 container2">
+                <img
+                  :src="UserInfo.photoURL"
+                  class="rounded-lg w-full"
+                  id="changePhoto"
+                  style="width:11rem; height:8rem; overflow:hidden"
+                />
+                <i
+                  class="bx bx-pencil"
+                  id="pencilImage"
+                  style="position:absolute;top:3.4rem;left:4.6rem;z-index:1000;color:white;font-size:4rem"
+                  @click="imageUploadClick()"
```

Comment:
I think you should set this to only be visible if hovered? Im not sure your call on that one